docker-compose.yml

```diff
@@ -1,20 +1,35 @@
 version: "3"
 services:
-  hapi-fhir-jpaserver-start:
+  fhir-jpaserver-start:
     build: .
-    container_name: hapi-fhir-jpaserver-start
+    container_name: fhir-jpaserver-start
```

Comment:
NABD, but it would be good to keep naming consistent, trying to call it "FHIR server" whenever I refer to it, so maybe `fhir-server`?